lldb/include/lldb/DataFormatters/TypeSynthetic.h

```diff
@@ -75,6 +77,10 @@ class SyntheticChildrenFrontEnd {
   // display purposes
   virtual ConstString GetSyntheticTypeName() { return ConstString(); }
 
+  virtual bool SetValueFromCString(const char *value_str, Status &error) {
+    return false;
+  }
+
```

Comment:
Where is this used? I assume you have code downstream that actually implements this?

Reply:
It will be used in the following patches (it is the first, two more to come). But maybe it is better to use one big patch with all changes?